requirements.txt

```diff
@@ -8,3 +8,5 @@ gunicorn>=20.0.4
 Flask-Caching>=1.3.3
 pyroclient@git+https://github.com/pyronear/pyro-api.git#egg=pyroclient&subdirectory=client
 python-dotenv>=0.15.0
+pycryptodome>=3.10.1
+geopy>=2.1.0
```

Comment:
I think that you forgot to add `simple-crypt>=4.1.7`

Reply:
I added a few lines in the README!